### ISSUE TITLE
GPU testing is temporarily unavailable

### ISSUE THREAD

**Issue by akihironitta** [PyTorchLightning/pytorch-lightning#11634]
## 🐛 Bug
The tests are not running on GPUs temporarily since last Friday as seen in https://dev.azure.com/PytorchLightning/pytorch-lightning/_build?definitionId=6&_a=summary.

### Additional context
We all are aware of this issue already but I'm creating this issue for tracking publicly.

cc @tchaton @rohitgr7 @akihironitta @carmocca @borda

**Comment by Borda** [comment]
GPU-CI us up and running in limited scope (max 8 agents in parallel)

**Comment by akihironitta** [comment]
Closing this issue as we no longer experience unavailability of GPU testing. Let's create another issue if there's any further planned improvements in GPU env in CI.
